oat-sa/tao-test-runner-qti-fe
# Fix/tca 551/test status review structure juml link hide review

**Related**
https://oat-sa.atlassian.net/browse/TCA-551

**Description**
"Test status and structure" jump link is present when Review panel is hidden in the test

**How to test**
1. Create the test with enabled review screen
2. Start the test as a test tacker
3. Check that "Test status and structure" jump link exists
4. Press hide review button check the JL again
5. Click the show review button and check the JL again
